### PR TITLE
[IMP] master-fifo-cost-inventory_adjustement-sed

### PR DESCRIPTION
**Pull request by sed-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock_account/models/stock.py [hunk 1]
```diff
@@ -369,10 +369,9 @@ def product_price_update_before_done(self, forced_qty=None):
         tmpl_dict = defaultdict(lambda: 0.0)
         # adapt standard price on incomming moves if the product cost_method is 'average'
         std_price_update = {}
-        for move in self.filtered(lambda move: move.location_id.usage in ('supplier', 'production') and move.product_id.cost_method == 'average'):
+        for move in self.filtered(lambda move: move._is_in() and move.product_id.cost_method == 'average'):
```
Comment (sle-odoo) [inline addons/stock_account/models/stock.py]: this should be in a separate commit

File: addons/stock_account/models/stock_inventory.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from odoo.tools.float_utils import float_compare
+
+
+class InventoryLine(models.Model):
+    _inherit = "stock.inventory.line"
+
+    cost_method = fields.Char(compute="_compute_cost_method")
```
Comment (sle-odoo) [inline addons/stock_account/models/stock_inventory.py]: related

**Comment by C3POdoo** [comment]
Dear @sed-odoo,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
